app/styles/ui/window/_tooltips.scss

```diff
@@ -160,4 +160,12 @@ body > .tooltip,
       margin-right: var(--spacing);
     }
   }
+
+  .title {
+    white-space: pre;
+  }
+
+  .description {
+    opacity: 0.7;
```

Comment:
Let's use our themed colors here instead of relying on opacity
```suggestion
    color: var(--text-secondary-color);
```

Reply:
I gave this a shot but it was extremely unreadable in light mode (where it's `$gray-500`). I also tried out out `$gray-400`, but that was still a bit darker than it should. And then `$gray-300`, but sadly there is a huge jump in brightness from 400 to 300 and that was barely distinguishable from white. I'd love something like `$gray-350`, but not having that, opacity 0.7 works equally well across dark and light modes, and is in the visibility/dimming sweet spot.

#### Original design
<img width="376" alt="135830820-98603c30-21c5-4027-8e31-f3c0748070a1" src="https://user-images.githubusercontent.com/4550621/145124131-482d2902-c94c-4f73-bee5-bcb96a284189.png">

#### Opacity 0.7
<img width="342" alt="Screen Shot 2021-12-08 at 01 10 33" src="https://user-images.githubusercontent.com/4550621/145125337-842ed099-761c-4778-b760-c2f4182f701e.png">

#### Gray 300
<img width="338" alt="Screen Shot 2021-12-08 at 00 47 21" src="https://user-images.githubusercontent.com/4550621/145124129-84601db4-1fe1-48dc-9134-37cf2f5450af.png">

#### Gray 400
<img width="346" alt="Screen Shot 2021-12-08 at 01 08 17" src="https://user-images.githubusercontent.com/4550621/145125112-d5285892-2b7c-4931-8f8b-d968fbe400f7.png">

#### Gray 500
<img width="351" alt="Screen Shot 2021-12-08 at 00 43 41" src="https://user-images.githubusercontent.com/4550621/145124135-9f513198-629f-4c15-a0e6-c9c3c94c947c.png">